services/wallet/service.go

```diff
@@ -246,6 +248,13 @@ func NewService(
 		collectiblesOwnershipController,
 		collectiblesPublisher)
 
+	// EFP (Ethereum Follow Protocol) providers
+	efpClient := efp.NewClient()
+	followingProviders := []efp.FollowingDataProvider{
```

Comment:
Check how we conditionally populate the other provider lists https://github.com/status-im/status-go/blob/develop/services/wallet/service.go#L119
Please do the same thing here, we don't want accesses to the provider if `thirdpartyServicesEnabled` is false